robot-command/src/main/java/org/obolibrary/robot/ExplainCommand.java

```diff
@@ -0,0 +1,146 @@
+package org.obolibrary.robot;
+
+import java.io.File;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Options;
+import org.semanticweb.owl.explanation.api.Explanation;
+import org.semanticweb.owlapi.apibinding.OWLManager;
+import org.semanticweb.owlapi.manchestersyntax.parser.ManchesterOWLSyntaxInlineAxiomParser;
+import org.semanticweb.owlapi.model.IRI;
+import org.semanticweb.owlapi.model.OWLAnnotationAssertionAxiom;
+import org.semanticweb.owlapi.model.OWLAxiom;
+import org.semanticweb.owlapi.model.OWLOntology;
+import org.semanticweb.owlapi.reasoner.OWLReasonerFactory;
+import org.semanticweb.owlapi.util.SimpleShortFormProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handles inputs and outputs for the {@link ExplainOperation}.
+ *
+ * @author <a href="mailto:balhoff@renci.org">Jim Balhoff</a>
+ */
+public class ExplainCommand implements Command {
+  /** Logger. */
+  private static final Logger logger = LoggerFactory.getLogger(ExplainCommand.class);
+
+  /** Store the command-line options for the command. */
+  private Options options;
+
+  public ExplainCommand() {
+    Options o = CommandLineHelper.getCommonOptions();
+    o.addOption("i", "input", true, "load ontology from a file");
+    o.addOption("I", "input-iri", true, "load ontology from an IRI");
+    o.addOption("r", "reasoner", true, "reasoner to use: ELK, HermiT, JFact");
+    o.addOption("a", "axiom", true, "the axiom to explain");
+    o.addOption("m", "max", true, "the maximum number of explanations to retrieve");
+    o.addOption("f", "format", true, "the explanation result format: MD, OFN, etc.");
+    o.addOption("o", "output", true, "save explanation to a file");
+    options = o;
+  }
+
+  @Override
+  public String getName() {
+    return "explain";
+  }
+
+  @Override
+  public String getDescription() {
+    return "explain derivation of an inferred axiom";
+  }
+
+  @Override
+  public String getUsage() {
+    return "robot explain --input <file> --reasoner <reasoner> --axiom <axiom> <output>";
```

Comment:
```
"robot explain --input <file> --axiom <axiom> --output <output>"
```
Default reasoner should is ELK (returned by `CommandLineHelper.getReasonerFactory(...)`) so the user wouldn't need to specify that. Maybe make a note in the docs? And if you link to the `reason` docs, they can see all available reasoners.